src/packages/generator-helper/src/dmmf.ts

```diff
@@ -75,11 +75,14 @@ export namespace DMMF {
     isList: boolean
     isUnique: boolean
     isId: boolean
+    isReadOnly: boolean
```

Comment:
@Jolg42 any idea about this?

Reply:
Looks like it's a thing, though I lack context where it's actually "true". Someone with Query Engine knowledge might know more.

Might be set to true on the virtual relations from what I can see, like https://www.prisma.io/docs/concepts/components/prisma-schema/relations

Reply:
Didn't find where this prop is set - but it surely is [in the tests](https://github.com/prisma/prisma/blob/master/src/packages/client/src/__tests__/__helpers__/dmmf-types.ts#L19)